server/vulnerabilities/oval/analyzer.go

```diff
@@ -0,0 +1,175 @@
+package oval
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	oval_parsed "github.com/fleetdm/fleet/v4/server/vulnerabilities/oval/parsed"
+)
+
+// Analyze scans all hosts for vulnerabilities based on the OVAL definitions for their platform.
```

Comment:
`Analyze` is also inserting into the provided `fleet.Datastore`. At first sight I thought it would use the `ds` to scan and then return the findings.